src/amo/components/SectionLinks/index.js

```diff
@@ -94,7 +95,8 @@ export class SectionLinksBase extends React.Component<Props> {
               'SectionLinks-link',
               'SectionLinks-link-theme',
               {
-                'SectionLinks-link--active': viewContext === ADDON_TYPE_THEME,
+                'SectionLinks-link--active':
+                  viewContext === ADDON_TYPE_STATIC_THEME,
```

Comment:
I believe we're not using `ADDON_TYPE_THEME` anymore **but** we still have lots of occurrences in the code base and we'll need to fix https://github.com/mozilla/addons-frontend/issues/8256 and https://github.com/mozilla/addons-frontend/issues/5359. I would suggest a slightly different fix:

```suggestion
			      // TODO: remove `ADDON_TYPE_THEME`
                  [ADDON_TYPE_THEME, ADDON_TYPE_STATIC_THEME].includes(viewContext),
```

Reply:
With this suggestion, I think the test case in `tests/unit/amo/components/TestSectionLinks.js` should pass. We could write a very similar test case for `ADDON_TYPE_STATIC_THEME`. (Yes, we need both)

Reply:
> With this suggestion, I think the test case in `tests/unit/amo/components/TestSectionLinks.js` should pass. We could write a very similar test case for `ADDON_TYPE_STATIC_THEME`. (Yes, we need both)

Thanks for the suggestion. I made the changes. 

Here's the test case for `ADDON_TYPE_STATIC_THEME` :

![Screenshot (233)](https://user-images.githubusercontent.com/21038543/62453927-05629b00-b791-11e9-911b-ed4eb7f122ef.png)
Let me know if the test title is correct.


> **but** we still have lots of occurrences in the code base and we'll need to fix #8256

and yeah, we have a some tests which are depending on the `ADDON_TYPE_THEME` and I think this is why I still left with 4 test suites failing there. I'll do that by this weekend.

Reply:
>  Let me know if the test title is correct.

Instead of `statictheme`, I would put the name of the constant. Or I would write:

```
... when add-on is a static theme

```

Reply:
> when add-on is a static theme

Okay, Thanks a lot. I've made the changes. :)